paddle/framework/tensor_array_test.cc

```diff
@@ -126,5 +126,57 @@ TEST_F(TensorArrayTester, size) {
   ASSERT_EQ(ta.size(), static_cast<size_t>(batch_size));
 }
 
+TEST(TensorArray, LodPack) {
+  // three time steps, each step stores a LoDTensors
```

Comment:
please add more explanation of this test.

Reply:
I will add in next PR